### PR TITLE
Improve primelib.py test coverage #9943 

### PR DESCRIPTION
**Pull request by rmp2000**
### Describe your change:
Improve the doctest in the primelib.py file.
* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: maths/primelib.py [hunk 1]
```diff
@@ -525,6 +726,17 @@ def simplify_fraction(numerator, denominator):
...
+
+    >>> simplify_fraction(10,20)
+    (1,2)
+    >>> simplify_fraction(10,-1)
+    Traceback (most recent call last):
+    ...
+    AssertionError: The arguments must been from type int and 'denominator' != 0"
+    >>> simplify_fraction("test","test")
+    Traceback (most recent call last):
+    ...
```
Comment (cclauss) [inline maths/primelib.py:738]:
```suggestion
    (1, 2)
    >>> simplify_fraction(10,-1)
    Traceback (most recent call last):
        ...
    AssertionError: The arguments must been from type int and 'denominator' != 0"
    >>> simplify_fraction("test","test")
    Traceback (most recent call last):
        ...
```

**Comment by cclauss** [comment]
There are git conflicts.  This means that someone else has modified this file.

**Comment by rmp2000** [comment]
> There are git conflicts. This means that someone else has modified this file.

is my firts time doing a pull request now what I sgould do ? thanks

**Comment by cclauss** [comment]
OK.  I merged the two changes together so we have two new tests plus your tests.

**Comment by rmp2000** [comment]
> OK. I merged the two changes together so we have two new tests plus your tests.

Nice, thanks for the help

**Review by cclauss — APPROVED** [review]
Great!  Thanks for doing this.

If you wanted to create a follow-up pull request to add `type hints` to function signatures like we did on line 794, that would be supercool!

**Pull request merged**